frontend/locale/en.json

```diff
@@ -9,28 +9,68 @@
   "404.message": "Sorry the page came back with a 404 error we can't find what you are looking for.",
   "footer.credit": "Created with {link}",
   "footer.icon_credit": "Icons made by {link} ({ccLink})",
```

Comment:
1. i'm not sure where to correct it, but i want to mention it:
the text of the **{ccLink}** is wrong, the license is abbreviated as "**(CC BY 3.0)**".

2. why make the whole thing longer than necessary - it becomes problematic on smaller screens (especially with longer translations):
wouldn't _"Icons by {link} ({ccLink})"_ be enough?

Reply:
Both our your points are correct. I'll make the changes.